frontend/src/tabs/authorship.pug

```diff
@@ -18,32 +19,32 @@
     .contribution(v-if="isLoaded && files.length!=0")
       .sorting.mui-form--inline
         .mui-select
-          select.medium-font(v-model="filesSortType")
+          select(v-model="filesSortType")
             option(value="lineOfCode") LoC
             option(value="path") Path
             option(value="fileName") File Name
             option(value="fileType") File Type
-          label.medium-font sort by
+          label sort by
         .mui-select.sort-order
-          select.medium-font(v-model="toReverseSortFiles")
+          select(v-model="toReverseSortFiles")
```

Comment:
this

Reply:
This is overwritten in the original css. Therefore, removed it.

Reply:
This is overwritten in the original css. Therefore, removed it.